kratos/sources/model_part.cpp

```diff
@@ -945,6 +945,36 @@ ModelPart::ElementType::Pointer ModelPart::CreateNewElement(std::string ElementN
     KRATOS_CATCH("")
 }
 
+/** Inserts an element in the mesh with ThisIndex.
+*/
+ModelPart::ElementType::Pointer ModelPart::CreateNewElement(std::string ElementName, 
```

Comment:
i would argue that we don t NEED this in c++, but i would agree having this in the python interface.

i am not much against anyway..